src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let currentSum = 0;
+
+  const adder = function (num) {
```

Comment:
This function can take any number of arguments, so using the ``num`` parameter limits it.
You can use arguments or the unpacking operator `(...args)`

Example:

```suggestion
  const adder = function (...args) {
```